Hermes/appscale/hermes/__init__.py

```diff
@@ -12,30 +10,30 @@
 import socket
 import sys
 import tarfile
+import urllib
+
 import tornado.escape
 import tornado.httpclient
 import tornado.web
-import urllib
-
 from appscale.common import appscale_info
 from appscale.common import appscale_utils
+from appscale.common.constants import LOG_FORMAT
 from appscale.common.ua_client import UAClient
 from appscale.common.ua_client import UAException
-from handlers import MainHandler
-from handlers import TaskHandler
-from helper import JSONTags
-
+from appscale.common.unpackaged import APPSCALE_PYTHON_APPSERVER
 from tornado.ioloop import IOLoop
 from tornado.ioloop import PeriodicCallback
-from tornado.options import define
 from tornado.options import options
-from tornado.options import parse_command_line
 
-sys.path.append(os.path.join(os.path.dirname(__file__), '../AppServer'))
+from appscale.hermes import constants
```

Comment:
@tmarballi , I guess you didn't refresh pull request page before reviewing. I did some changes to address Chris's comments and imports was changed too.
I could miss some module, but generally I followed practice of splitting imports into 3 sections, each section is alphabetically ordered. Build-in imports go first, next are third party (modules which don't belong to this project, so appscale.common is here), after that project imports are (appscale.hermes ...).
Import from google.appengine.api goes after other imports because it uses manual python path.